Mono.Debugging/Mono.Debugging.Client/DebuggerSession.cs

```diff
@@ -49,6 +49,8 @@ public abstract class DebuggerSession: IDisposable
 	{
 		readonly Dictionary<BreakEvent, BreakEventInfo> breakpoints = new Dictionary<BreakEvent, BreakEventInfo> ();
 		readonly Dictionary<string, string> resolvedExpressionCache = new Dictionary<string, string> ();
+		public List<Assembly> assemblies = new List<Assembly> ();
```

Comment:
This shouldn't be part of the public API.
```suggestion
	List<Assembly> assemblies = new List<Assembly> ();
```